lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py

```diff
@@ -70,6 +77,21 @@
     timeout: 10
     exact_match: no
     remove_duplicate: yes
+
+- name: Specify host inventory name and get host info
```

Comment:
Same as above, description is not very clear:
```yaml
- name: Reduce host inventory information to provided keys
```

Reply:
Thank you.
I will modify task name of examples.